packaging/components.nix

```diff
@@ -12,6 +11,15 @@ let
   inherit (scope)
     callPackage
     ;
+  inherit
+    (scope.callPackage (
+      { stdenv }:
+      {
+        inherit stdenv;
+      }
+    ) { })
+    stdenv
+    ;
```

Comment:
Not pretty. Will consider moving the build utilities to their own file, and then `callPackage` that.

Reply:
Either way we aren't making much use of the two layers of scoping, and we could simplify by squashing those into one scope. I'd keep the separate file for dependencies, but wire it up with `import` and `//` - should be simple enough.